docs/UserGuide.md

```diff
@@ -362,6 +362,12 @@ up instead.
 
 </box>
 
+<box type="warning" seamless>
+
+Note that the backslash `\` is **not** allowed to be inputted as part of any parameter as it is reserved to signify the prefixes.
```

Comment:
Is there a better way to phrase this? Perhaps "The backslash character \ is reserved and should not be included as part of any parameter input."

Reply:
Changed to

> Note that the backslash `\` is reserved for prefixes. It is **not** allowed to be part of any inputted parameter.